app/experimenter/nimbus-ui/src/components/FormBranches/index.tsx

```diff
@@ -0,0 +1,228 @@
+            <Form.Label>
+              Branch{" "}
+              {isReference && (
+                <Badge pill variant="primary" data-testid="control-pill">
+                  control
+                </Badge>
+              )}
+            </Form.Label>
+            <Form.Control type="text" defaultValue={name} />
+          </Form.Group>
+          <Form.Group as={Col} controlId="description">
+            <Form.Label>Description</Form.Label>
+            <Form.Control type="text" defaultValue={description} />
+          </Form.Group>
+          <Form.Group as={Col} controlId="ratio" sm={2} md={2}>
+            <Form.Label>Ratio</Form.Label>
+            {equalRatio ? (
+              <p data-testid="equal-ratio" className="p-0 m-0">
+                Equal
+              </p>
+            ) : (
+              <Form.Control type="text" defaultValue={ratio} />
+            )}
+          </Form.Group>
+          <Form.Group as={Col} sm={1} className="align-top text-right">
+            {!isReference && (
+              <Button
+                variant="light"
+                className="bg-transparent border-0 p-0 m-0"
+              >
+                <DeleteIcon width="18" height="18" />
+              </Button>
+            )}
+          </Form.Group>
+        </Form.Row>
+      </Form.Group>
+
+      {featureValue === null || !experimentFeatureConfig ? (
+        <Form.Group className="px-2 mx-3">
+          <Form.Row>
+            <Button
+              variant="outline-primary"
+              size="sm"
+              data-testid="feature-config-add"
+            >
+              + Feature configuration
+            </Button>
+          </Form.Row>
+        </Form.Group>
+      ) : (
+        <Form.Group
+          className="px-3 py-2 border-top"
+          data-testid="feature-config-edit"
+        >
+          <p>Feature configuration</p>
+          <Form.Row className="align-middle">
+            <Form.Group as={Col} controlId="feature">
+              <Form.Control
+                as="select"
+                defaultValue={experimentFeatureConfig.slug}
+              >
+                <option value="">Select...</option>
+                {featureConfig &&
+                  featureConfig.map(
+                    (feature, idx) =>
+                      feature && (
+                        <option
+                          key={`feature-${feature.slug}-${idx}`}
+                          value={feature.slug}
+                        >
+                          {feature!.name}
+                        </option>
+                      ),
+                  )}
+              </Form.Control>
+            </Form.Group>
+            <Col sm={1} md={1} className="px-2 text-center">
+              is
+            </Col>
+            <Form.Group as={Col} controlId="featureEnabled">
+              <Form.Check
+                type="switch"
+                label={featureEnabled ? "On" : "Off"}
```

Comment:
In local testing this doesn't appear to change to "Off" when disabled
<img width="134" alt="Screen Shot 2020-10-30 at 7 23 38 PM" src="https://user-images.githubusercontent.com/6392049/97764573-6f2d3a00-1ae5-11eb-94db-b133a2d69d32.png">

Reply:
Yeah, that's part of the interaction handling to follow in EXP-505 / issue #3781